bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -25,7 +25,8 @@ limitations under the License.
   <artifactId>bigtable-beam-import</artifactId>
 
   <properties>
-    <mainClass>com.google.cloud.bigtable.beam.sequencefiles.Main</mainClass>
+    <mainClass>com.google.cloud.bigtable.beam.Main</mainClass>
+    <skipITs>false</skipITs>
```

Comment:
I don't think we should be changing this, ITs should be activated via profile.

Reply:
Sure, will revert this.